README.md

```diff
@@ -1,6 +1,12 @@
 # container-helm
 
-[![Docker Build Status](https://img.shields.io/docker/build/appuio/helm.svg)](https://hub.docker.com/r/appuio/helm/)
+[![dockeri.co](http://dockeri.co/image/appuio/helm)](https://hub.docker.com/r/appuio/helm/)
+
+[![Build Status](https://img.shields.io/docker/cloud/build/appuio/helm.svg)](https://hub.docker.com/r/appuio/helm/builds
+) [![GitHub issues](https://img.shields.io/github/issues-raw/appuio/container-helm.svg)](https://github.com/appuio/container-helm/issues
```

Comment:
Why is the closing bracket `)` always on the new line? Markdown will put items on consecutive lines (in sourcecode) on the same line when rendered...

Reply:
This is exactly to solve the issue with Markdown (and Docker Hub flavored Markdown), which adds a line break for every single line break it finds. When you do this inside a link the line break will not happen and the link will still be rendered correctly.

Reply:
... and this way we can cleanly split up the lines for both better readability and easier modification (e.g. add or remove lines, showing appropriate changes in Git). It's like placing list entries in Puppet with commas on single lines.

Reply:
> Markdown will put items on consecutive lines (in sourcecode) on the same line when rendered...

Only on GitHub, not on Docker Hub. Unfortunately!

Reply:
Thanks for clarification, it was not clear to me whether this was on purpose or by accident.

> Only on GitHub, not on Docker Hub. Unfortunately!
Ah , I was not aware of that